AEM/getting-started-with-automixedprecisionmkl.md

```diff
@@ -199,15 +161,27 @@ python conv2D_bf16.py
 2022-08-19 00:06:08.750799: I tensorflow/core/grappler/optimizers/auto_mixed_precision.cc:2303] Running auto_mixed_precision_onednn_bfloat16 graph optimizer
 2022-08-19 00:06:08.751058: I tensorflow/core/grappler/optimizers/auto_mixed_precision.cc:2240] Converted 3/11 nodes to bfloat16 precision using 0 cast(s) to bfloat16 (excluding Const and Variable casts)
 ```
-The data-flow graph after porting the model to BFloat16 type looks like below.
+> _No allowlist ops found typically refers to the data session._
+> One can also use ONEDNN_VERBOSE logs as discussed earlier to verify AMX BF16 instructions.
 
-![amp-mkl2](/content/dam/develop/external/us/en/images/automixedprecisionmkl-2.png)
+## IV. Tips for best performance on Intel® Xeon CPUs
+### Using [numactl](https://man7.org/linux/man-pages/man8/numactl.8.html)
+One should use numactl to run your code to obtain best performance. 
+> Following are some examples and one can tweak the numactl parameters to get best performance for their specific usecase.
 
-Notice that 2 operators, Conv2D+BiasAdd and ReLU in the graph are automatically converted to operate in BFloat16 type. Also note that appropriate Cast nodes are inserted in the graph to convert TensorFlow* tensors from FP32 type to BFloat16 type and vice-versa.
+#### 1. for best throughput usecase performance, try using the whole socket or as many cores as posible
+```bash
+numactl --localalloc --cpunodebind 0 python conv2D_bf16.py
+```
+#### 2. for best latency usecase performance, try using fewer cores
+```bash
+numactl --localalloc --physcpunodebind=0-3 python conv2D_bf16.py
+```
 
- 
-### Steps to Convert a Pretrained fp32 Model to BFloat16
-In the previous section we saw how the AutoBFloat16Convertor can automatically convert certain nodes to BFloat16 while training a sample model. This section will cover how to convert a pre-trained fp32 model to BFloat16.
+## Appendix 1 : Steps to convert a Pretrained fp32 Model to BFloat16
```

Comment:
Lets remove this section also. We don't want people to use this, they should dynamically set auto_mixed.

Reply:
yes, will remove it. Seems too much details.